Sunrise/Models/Attribute.swift

```diff
@@ -20,4 +20,27 @@ struct Attribute: Mappable {
         value              <- map["value"]
     }
 
+    // MARK: - Value string representation
+
+    func value(type: AttributeType) -> String? {
+        guard let typeName = type.name else { return nil }
+
+        switch (typeName, value) {
+            case ("boolean", let value as Bool):
+                return value ? NSLocalizedString("Yes", comment: "Yes") : NSLocalizedString("No", comment: "No")
+            case (let typeName, let value as String) where ["text", "enum"].contains(typeName):
+                return value
+            case (let typeName, let value as [String: String]) where ["ltext", "lenum"].contains(typeName):
+                return value.localizedString
+            case ("number", let value as Int):
+                return String(value)
+            case ("number", let value as Double):
+                return String(value)
+            case ("money", let value):
+                return Mapper<Money>().map(value)?.description
+
```

Comment:
From the plain types, I think `Date`, `Time` and `DateTime` are missing.

We also have the `Reference` type, but I'm not sure whether this can be handled in a generic way - I'd still think it's good to add a comment that this is ignored.

Furthermore there is the `Set` type. I think it's worth adding it (if you prefer, we can also do that in a different PR).

Finally, we have the `Nested` type. Like for reference, I'd probably vote to ignore it for now, but still add a comment about it.